ext/src/utils.rs

```diff
@@ -572,9 +614,31 @@ mod timer {
 
         pub fn end(self, _msg: std::fmt::Arguments) {}
     }
+
+    #[repr(transparent)]
```

Comment:
Why is this needed?

Reply:
Just to make sure the newtype is really zero-cost. It's probably unnecessary, but I figured why not make sure